server/config/routes.js

```diff
@@ -26,6 +26,8 @@ module.exports.routes = {
 
   // Forms APIs
   'get /api/forms/get-form': 'forms/get-form',
+  'get /api/forms/get-signedup-form': 'forms/get-signedup-form',
+  'get /api/forms/get-unsignedup-form': 'forms/get-unsignedup-form',
```

Comment:
also you need to add these apis in `policies.js` and specify the access rights. should be ['isLoggedIn'] i believe